### PR TITLE
Fix #144: Executable war

### LINKED ISSUE #144: Executable war
Add embedded tomcat as the provided dependency (to be part of `WEB-INF/lib-provided` but not `lib`) and make it possible to run it via `java -jar`.

### PR DESCRIPTION
**Pull request by jnpsk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,6 +96,12 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-validation</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.tomcat.embed</groupId>
+                    <artifactId>tomcat-embed-el</artifactId>
```
Comment (banterCZ) [inline pom.xml:102]: Is `tomcat-embed-el` present in `provided-lib`? I am afraid that this excludes the dependency completely. I made the same mistake and tried to fix it in this way. https://github.com/wultra/powerauth-server/pull/1087/files

**Review by banterCZ — COMMENTED** [review]
Please also double-check the configuration of the `enforcer-plugin`. See https://github.com/wultra/powerauth-server/blob/develop/pom.xml#L223

File: pom.xml (hunk 1 above)
Comment (jnpsk) [reply pom.xml:102]: I checked the content of the `provided-lib` once more and the it really contains `tomcat-embed-{core,el,websocket}`.

Comment (banterCZ) [reply pom.xml:102]: Yes, you are right, it comes from `spring-boot-starter-tomcat`. So it is up to you, whether to do it via `exclusion` or `dependency-management`, but either way, the enforcer plugin will validate that it is not a part of `WEB-INF/lib`.

**Review by banterCZ — CHANGES_REQUESTED** [review]
Perfect!
I am requesting changes just to block the merge until **release 1.5.0**

**Pull request merged**